routers/api/v1/api.go

```diff
@@ -593,17 +593,15 @@ func RegisterRoutes(m *macaron.Macaron) {
 		}, reqToken())
 
 		// Repositories
-		m.Post("/org/:org/repos", reqToken(), bind(api.CreateRepoOption{}), repo.CreateOrgRepo)
+		m.Post("/org/:org/repos", reqToken(), bind(api.CreateRepoOption{}), repo.CreateOrgRepoDeprecated)
+
+		m.Combo("/repositories/:id", reqToken()).Get(repo.GetByID)
 
 		m.Group("/repos", func() {
 			m.Get("/search", repo.Search)
-		})
```

Comment:
Are there the same group middlewares between these two group?

Reply:
same end here : [link](https://github.com/go-gitea/gitea/pull/9186/files#diff-2620fdbeeb83ce43692534e6c2c39452R803)

so there is NO reqXY in both of these Groups

Reply:
@lunny or do you mean something else?!?